trip_app/trips/tests/trip_factory.py

```diff
@@ -1,35 +1,41 @@
 from datetime import timedelta
 
 import factory
-from accounts.tests.user_factory import UserFactory
 from django.contrib.gis.geos import Point
 from django.utils import timezone
 from faker import Factory as FakerFactory
 
-from ..models import Trip
+from accounts.tests.user_factory import UserFactory
+from trips.models import Trip
 
 faker = FakerFactory.create()
 
 
 class TripFactory(factory.django.DjangoModelFactory):
     """Trip factory."""
+
     driver = factory.SubFactory(UserFactory)
     dep_time = timezone.now() + timedelta(days=1)
-    start_point = Point(float(faker.latitude()), float(faker.longitude()), srid=4326)
-    dest_point = Point(float(faker.latitude()), float(faker.longitude()), srid=4326)
+    start_point = Point(
+        float(faker.latitude()), float(faker.longitude()), srid=4326
```

Comment:
And here you cast both - lat and long to float

Reply:
Here I have to cast in both cases as  `faker.latitude()` returns Decimal object but the initializer of Point class expect it to be only `int` or `float` and if I didn't do the cast, the TypeError will be raised.